### PR TITLE
Fix duplication of save_config_to_disk and load_config_to_runtime function

### LINKED ISSUE #68: refactor save_config_to_disk() and load_config_to_runtime()
the functions `save_config_to_disc()` and `load_config_to_runtime()` are used in every module.   
better refactor them and move them also to `plugins/module_utils/mysql.py`

### PR DESCRIPTION
**Pull request by laurent-indermuehle**
fixes https://github.com/ansible-collections/community.proxysql/issues/68

### CODE REVIEW

File: changelogs/fragments/68-refactor-save-load-config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - module_utils - Refactor save_config_to_disk and load_config_to_runtime
```
Comment (Andersson007) [inline changelogs/fragments/68-refactor-save-load-config.yml:3]:
```suggestion
  - module_utils - Refactor save_config_to_disk and load_config_to_runtime.
```

**Review by Andersson007 — COMMENTED** [review]
@laurent-indermuehle thanks for the PR!

I'll take a closer look a bit later

cc @markuman

File: changelogs/fragments/68-refactor-save-load-config.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
```
Comment (Andersson007) [inline changelogs/fragments/68-refactor-save-load-config.yml:4]:
```suggestion
```

Comment (Andersson007) [reply changelogs/fragments/68-refactor-save-load-config.yml:4]: this should fix sanity failings @laurent-indermuehle ^

File: changelogs/fragments/68-refactor-save-load-config.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - module_utils - Refactor save_config_to_disk and load_config_to_runtime.
```
Comment (markuman) [inline changelogs/fragments/68-refactor-save-load-config.yml:3]:
```suggestion
  - module_utils - Refactor save_config_to_disk and load_config_to_runtime (https://github.com/ansible-collections/community.proxysql/pull/78).
```

add link to the pull request.

**Comment by Andersson007** [comment]
(Needless to day but please ignore the codecov's failings)

File: changelogs/fragments/68-refactor-save-load-config.yml (hunk 3 above)
Comment (Andersson007) [reply changelogs/fragments/68-refactor-save-load-config.yml:3]: @markuman ++

**Review by markuman — CHANGES_REQUESTED** [review]
just the missing pr url in the changelog fragment. otherwise looks good to me.

**Comment by laurent-indermuehle** [comment]
I'm lost. I think I merged the link to this PR in the changelog/fragment. But GitHub still says there is a change requested. When I click on [See review](https://github.com/ansible-collections/community.proxysql/pull/78#pullrequestreview-766979728) I see the changelog without the link... but in the comment, I can see the link. What did I do wrong?

**Comment by Andersson007** [comment]
> I'm lost. I think I merged the link to this PR in the changelog/fragment. But GitHub still says there is a change requested. When I click on [See review](https://github.com/ansible-collections/community.proxysql/pull/78#pullrequestreview-766979728) I see the changelog without the link... but in the comment, I can see the link. What did I do wrong?

@laurent-indermuehle that's fine:) If the request author approves the PR, this will disappear

File: plugins/modules/proxysql_global_variables.py [hunk 4]
```diff
@@ -156,9 +142,9 @@ def set_config(variable, value, cursor):
 def manage_config(variable, save_to_disk, load_to_runtime, cursor, state):
     if state:
         if save_to_disk:
-            save_config_to_disk(variable, cursor)
+            save_config_to_disk(cursor, "VARIABLES", variable)
```
Comment (laurent-indermuehle) [inline plugins/modules/proxysql_global_variables.py:145]: To be honest, I'm not sure to understand what the variable "variable" contains.
I think it is the name of the variable we are saving/loading.

But, why is this only used in this module? I don't know. Maybe there is more room for refactoring.

Comment (Andersson007) [reply plugins/modules/proxysql_global_variables.py:145]: @laurent-indermuehle yeah, feel free to refactor it more after this PR gets merged! Also there's a good reason to learn `pytest` for unit testing:)
you can find many examples in other collections, e.g. in community.mysql. Though be careful, I'm not sure my fake cursor class implementation is OK in general and adheres best practices;)

**Review by Andersson007 — APPROVED** [review]
LGTM

**Comment by Andersson007** [comment]
as @markuman approved the rest, I'll merge this.

@laurent-indermuehle thanks for the contribution!
@markuman thanks for reviewing!

**Pull request merged**